ideascube/migrations/0009_add_a_system_user.py

```diff
@@ -6,8 +6,9 @@
 
 
 def add_user(*args):
-    User = get_user_model()
-    User(serial='__system__', full_name='System', password='!!').save()
+    queryset = get_user_model().objects.get_queryset_unfiltered()
+    queryset.update_or_create(serial='__system__',
+                              defaults={'full_name':'System', 'password':'!!'})
```

Comment:
I have the same remark about `get_queryset_unfiltered` here.

However, thinking about it some more, I don't like this any better:

```python
    get_user_model().objects.all(include_system_user=True).update_or_create(...)
```

So maybe we should just drop the `.all(include_system_user=True)` method, and just use `get_queryset_unfiltered()` everywhere. (it's currently used in exactly one place in the code and in one test)

Not something for this pull request, but something to keep in mind.